website/pages/docs/concepts/storage.mdx

```diff
@@ -0,0 +1,74 @@
+---
+layout: docs
+page_title: Storage
+sidebar_title: Storage
+description: >-
+  Vault relies on external storage to save its information.
+---
+
+# Storage
+
+As described on our [Architecture](/docs/internals/architecture) page, Vault's storage backend is untrusted storage used purely to keep encrypted information.
+
+## Supported Storage Backends
+
+For enterprise customers, HashiCorp provides official support for Consul and Vault's Integrated Storage as storage backends.
+
+Many other options for storage are available with community support - see our [Storage Configuration](/docs/configuration/storage) section for more information.
+
+## Backups & Restores
+
+Due to the highly flexible nature of Vault's potential storage configurations, providing exact guidance on backing up and restoring Vault is challenging.
+
+When backing up Vault, there are two pieces to consider:
+
+1. Vault's encrypted data in the storage backend
+2. Configuration files and management scripts for running the Vault server
+
+There's also a big question - what is the error case you're trying to guard against by saving a backup?
+
+### The Big Question - Why Take Backups?
+
+It's important to consider the question of "why take a backup" while developing your ongoing
+backup strategy.
+
+We do not recommend backups as protection against the failure of an individual machine or data
+center. Vault servers can run in clusters. Vault Enterprise supports replicated clusters for
+performance and disaster recovery. With open source features, a Vault cluster can extend across
+multiple availability zones within a region.
+
+As you develop a plan for backing up and restoring Vault, include clustering in your plan to
+address redundancy concerns.
+
+### Vault's Persisted Data
+
+Backups and restores are ideally performed while Vault is offline. If offline backups are not
+feasible, we recommend using a storage backend that supports atomic snapshots (such as
+[Consul](https://www.consul.io/docs/commands/snapshot.html) or [Integrated
+Storage](/docs/commands/operator/raft#snapshot)).
+
+**If your storage backend does not support atomic snapshots, we recommend only taking offline backups.**
+
+We recommend taking backups **before**, but not during, write operations to the `/sys` API
+(excluding the `/sys/leases`, `/sys/namespaces`, `/sys/tools`, `/sys/wrapping`, `/sys/policies`, and
+`/sys/pprof` endpoints). Some examples of workflows that write to the `/sys` API are upgrades,
+server moves, and rekeys. In the future, this guidance may change for the Integrated Storage
```

Comment:
Is this referring to replication state changes, or does it entail more than that?

Reply:
I read it as changes to raft peers.

Reply:
I believe I copied it from someone's suggestion (which may have been on the original google doc), and so am not certain what it means and will remove it.